presto-main/src/main/java/io/prestosql/operator/aggregation/minmaxby/AbstractMinMaxBy.java

```diff
@@ -280,4 +303,14 @@ private void generateOutputMethod(ClassDefinition definition, CallSiteBinder bin
         ifStatement.ifFalse(valueSqlType.writeValue(out, getValueExpression));
         method.getBody().append(ifStatement).ret();
     }
+
+    private static Method getMethod(Class<?> stateClass, String name)
+    {
+        for (Method method : stateClass.getMethods()) {
```

Comment:
This could be
```java
return stream(stateClass.getMethods())
        .filter(method -> method.getName().equals(name))
        .findFirst()
        .orElseThrow(() -> new IllegalArgumentException("State class does not have a method named " + name));